zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -38,7 +42,18 @@ def run(self, lines: List[str]) -> List[str]:
                 endpoint, method = doc_name.rsplit(':', 1)
                 return_values: Dict[str, Any] = {}
                 return_values = get_openapi_return_values(endpoint, method)
-                text = self.render_table(return_values, 0)
+                text: List[str] = []
+                if endpoint != '/events':
+                    text = self.render_table(return_values, 0)
+                else:
+                    events = return_values['events'].pop('items', None)
+                    text = self.render_table(return_values, 0)
+                    # Another heading for the events documentation
+                    text.append('\n\n## Events\n\n')
+                    text += self.render_events(events)
+                    # We have modified the openapi_spec object
+                    # so it should be reloaded
+                    openapi_spec.reload()
```

Comment:
This hack would be better replaced by using `.deepcopy` and modifying the copy, I think, rather than modifying the source data and reloading it.  I'm concerned the reload approach could result in subtle caching/multithreading problems.

Reply:
Done.